rust-maven-example/src/main/java/io/questdb/example/rust/Main.java

```diff
@@ -29,7 +29,7 @@
 public class Main {
     public static native String reversedString(String str);
 
-    static {
+    public static void main(String[] args) {
```

Comment:
I think this should stay in a `static` block as it's generally how JNI libs are loaded.
No need to change this code.